spec/support/environment_spec_helper.rb

```diff
@@ -1,6 +1,8 @@
 module EnvironmentSpecHelper
   # @todo rewrite using with_env_vars
   def with_18f_procurement_env_variables
+    ENV['DISABLE_SANDBOX_WARNING'] = 'true'
```

Comment:
Nevermind, took care of it in #625. Whichever gets merged first should tweak this though.